### PR TITLE
do not update a submodule if needed

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

**Comment by Gustry** [comment]
Same as #15, my PR comes from another repo, the transifex token is not set in travis

**Comment by 3nids** [comment]
can you explain a bit the issue this fixes?

**Comment by Gustry** [comment]
We are running qgis-plugin-ci on our private gitlab with some submodules which are private too. The gitlab-ci will manage before how to fetch submodules in the current job. I understand, this PR is quite specific, but it's needed for us.

**Comment by 3nids** [comment]
Ok thanks for the explanation. Some minor phrasing changing and I'll merge this. 
Have you tested it already?
Cheers

**Comment by Gustry** [comment]
> Some minor phrasing changing and I'll merge this.

I can but which one?

> Have you tested it already?

Yes, there for instance https://github.com/3liz/qgis-gobs-plugin/blob/master/.gitlab-ci.yml#L13
And other private repo.

**Comment by 3nids** [comment]
See the 2 comments above

**Comment by Gustry** [comment]
You haven't validated the review I guess ;-)

File: README.md [hunk 1]
```diff
@@ -72,6 +75,9 @@ optional arguments:
                         releasing. If specified and some changes are detected,
                         a hard reset on a stash create will be used to revert
                         changes made by qgis-plugin-ci.
+  --disallow-submodule-update
```
[the remark this answers is not in the document]
Comment (Gustry) [reply README.md:78]: I totally agree, but somehow, at that time, I wanted to keep the same pattern with `--allow-uncommitted-changes`. Don't ask me why ... :) Let's change!

**Comment by Gustry** [comment]
Can you release a new version after this PR?
So I can try all my latest merged PR. Thanks

**Pull request merged**